src/test/java/seedu/address/storage/client/JsonClientManagerStorageTest.java

```diff
@@ -48,16 +48,19 @@ public void read_missingFile_emptyResult() throws Exception {
     @Test
     public void read_notJsonFormat_exceptionThrown() {
         assertThrows(DataConversionException.class, () -> readAddressBook("notJsonFormatClient.json"));
+        assertThrows(DataConversionException.class, () -> readAddressBook("notJsonFormatClient.json"));
```

Comment:
Do we need 2 lines of the same assertion?

Reply:
Lines 57, 63 and 111 as well

Reply:
Thx for the catch! Removed them :smile: